src/components/Navigation/FoodCategories/FoodCategories.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+  <ul class="np-category-list" @click="setActiveCategory">
+    <li class="np-category-list__item"
+        v-for="category in categories"
+        @click="setActiveCategory"
+        :id="category.type"
+        :class="activeCategory === category.type ? 'np-category-list__item--active' : ''"
+        :key="category.name">
+
+      <img class="np-category-list__item-icon"
+           :src="category.src"
+           :alt="altText(category.name)" />
+
+      <span class="np-category-active">{{ category.name }}</span>
+    </li>
+  </ul>
+</template>
+
+<script lang="ts">
+import { defineComponent, ref } from "vue";
+
+export default defineComponent({
+  props: {
+    categories: {
+      type: Array
+    },
+    activeCat: {
+      type: String,
+      default: "all"
+    //  zrobić default all z ikoną restauracji to będzie pokazywać wszystkie opcje
```

Comment:
if you type //todo in comment then you can browse all comments like that more easly in your IDE